repo-client/src/main/java/io/lionweb/repoclient/impl/ClientForBulkAPIs.java

```diff
@@ -31,18 +33,18 @@ public ClientForBulkAPIs(RepoClientConfiguration repoClientConfiguration) {
   }
 
   @Override
-  public void createPartitions(List<Node> partitions) throws IOException {
-    createPartitions(
+  public @Nullable Long createPartitions(List<Node> partitions) throws IOException {
```

Comment:
Maybe add some JavaDoc what the return value means?

If I get it right, it means the version of the history. In that case, I'd create an interface that clearly represents that meaning, and have potentially different implementations -- one repo might just count them, another might use git hashes, etc.

Reply:
I agree that having a type that more clearly represents the repository version makes sense, so I introduced `RepositoryVersionToken`.

However, I think that an interface with multiple implementations would be overkill at the current stage. I think we can just treat the version as a string. It should work for all kind of version tokens used. I think we can ignore the fact that htis particular implementation of the repository returns a long and just treat it as string, as we do not expect to do math on it or even sort the versions specified.

Reply:
So we don't want `RepositoryVersionToken` to be `Comparable`? I don't have an issue with that, we just should make a _conscious_ decision about that.

Reply:
I would start by _not_ making it `Comparable`, as if someone start using things other than longs for identifying versions I would not know how to compare them. If someone signals the needs to have versions comparable, then we can add it